core/architecture.md

```diff
@@ -0,0 +1,82 @@
+# Nimbess Controller Architecture
+
+## Overview
+
+Nimbess is a light-weight distributed controller architecture that runs as a
+Kubernetes pod across every Kubelet node. Nimbess by default supports the
+Berkley Extensible Software Switch [BESS](https://github.com/NetSys/bess)
+dataplane in order to achieve maximum network pipeline programmability and
```

Comment:
Today we think BESS has the best performance, but can we truly say that it is? Or should we have real details why we think BESS is better?